bin/cli.js

```diff
@@ -37,7 +37,7 @@ function exitSuccessfully(text) {
 function initSailsMigrations(env) {
   if (!env.modulePath) {
     console.log(chalk.red('No local sails_migrations install found in:'), chalk.magenta(tildify(env.cwd)));
-    exit('Try running: `npm install sails-migrations`');
+    process.exit('Try running: `npm install sails-migrations`');
```

Comment:
although this is only part of the error, since you shouldnt even reach to this situation

but you need to call `exitWithError` function